event-emitter/lib/index.js

```diff
@@ -1,12 +1,18 @@
-const Server = require('./server');
+'use strict';
 
-if (process.env.NODE_ENV !== 'test') {
-  const server = Server();
+const MyEventEmitter = require('./emitter');
+
+function run() {
+  const emitter = new MyEventEmitter();
+
+  // Log `tick` events. There can be multiple subscribers.
+  emitter.on('tick', ({ count }) => console.log(count));
 
-  server
-    .listen()
-    .then(() => console.log(`Listening on ${server.port}`))
-    .catch(() => console.log('port in use'));
+  // Shutdown the emitter after 5s.
+  return new Promise(resolve => setTimeout(resolve, 5000))
+    .then(() => emitter.shutdown());
 }
 
-module.exports = require('../native/index.node');
+if (process.env.NODE_ENV !== 'test') {
+  run();
+}
```

Comment:
```suggestion
}

module.exports = run;
```

Reply:
To keep this example consistent with all the other examples, we need to export run from `./lib/index.js`